coala_quickstart/coala_quickstart.py

```diff
@@ -32,6 +59,13 @@ def main():
     print_used_languages(printer, used_languages)
 
     relevant_bears = filter_relevant_bears(used_languages)
+
+    if args.ci:
+        for section, bears in relevant_bears.items():
+            relevant_bears[section] = [
+                bear for bear in bears
+                if not bear.get_non_optional_settings()]
```

Comment:
@sils Questions will be asked if there are non-optional settings.

@jayvdb I think a better way to do this would be to pass a flag to `filter_relevant_bears` and not touch them here?